lkcrypto.c

```diff
@@ -298,10 +300,12 @@ static int cache_warm(int tfm_type, struct hash_lock_t *state,
 
 static int hasher_cache_warm(void *arg)
 {
+	wait_for_completion_killable(&hasher_needs_init);
```

Comment:
If these return nonzero it means we got killed, so we should exit instead of entering `cache_warm`.

Reply:
Addressed